scripts/services/dependencies.ts

```diff
@@ -72,6 +72,7 @@ const checks: Array<Check> = [
     "getrawtransaction",
     "getpeerinfo",
     "getmempoolinfo",
+    "getzmqnotifications",
```

Comment:
What's this for?

Reply:
Ah, should have read your comment first. Do you know why getzmqnotifications is needed by 0.15?

Reply:
no idea. but proxy showed LND trying to invoke it in the logs once I upgraded to v15.